server/githookHandler.go

```diff
@@ -17,30 +18,43 @@ import (
 	"github.com/google/uuid"
 )
 
-// TransformListen converts a Github event into an Eventgrid event and publishes the event to Eventgrid
-func TransformListen(c buffalo.Context) error {
-
+// GithookReceiver converts a Github event into an Eventgrid event and publishes the event to Eventgrid
+func GithookReceiver(c buffalo.Context) error {
+	webhookSecret := os.Getenv("webhookSecret");
+	eventgridSecret := os.Getenv("eventgridSecret");
+	eventgridEndpoint := os.Getenv("eventgridEndpoint");
```

Comment:
It's more typical in a buffalo application to use the [`envy`](https://github.com/gobuffalo/envy) package than to use the standard library to fetch environment variables. It has at least one big advantage, which is that it automatically reads in `.env` files and adds them to the environment at application start up.

Reply:
Somewhat unrelated, for my non-buffalo projects, I'm a big fan of [`viper`](https://github.com/spf13/viper) for configuration management.